src/Uno.Templates/content/unoapp/.template.config/template.json

```
@@ -408,6 +408,39 @@
         }
       ]
     },
+    "authentication": {
+      "displayName": "Authentication",
+      "description": "Configures authentication support in the application",
+      "type": "parameter",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "none",
+          "description": "No authentication",
+          "displayName": "None"
+        },
+        {
+          "choice": "custom",
+          "description": "Application uses custom logic to authenticate user",
+          "displayName": "Custom"
+        }
+        // {
+        //   "choice": "msal",
+        //   "description": "Application uses MSAL to authenticate user",
+        //   "displayName": "MSAL"
+        // },
+        // {
+        //   "choice": "oidc",
+        //   "description": "Application uses OpenID Connect to authenticate user",
+        //   "displayName": "OIDC"
+        // },
+        // {
+        //   "choice": "Web",
+        //   "description": "Application uses web based authentication to authenticate user",
+        //   "displayName": "Web"
+        // }
```

may want to make this easier for your future PRs
```suggestion
        // },
        // {
        //   "choice": "msal",
        //   "description": "Application uses MSAL to authenticate user",
        //   "displayName": "MSAL"
        // },
        // {
        //   "choice": "oidc",
        //   "description": "Application uses OpenID Connect to authenticate user",
        //   "displayName": "OIDC"
        // },
        // {
        //   "choice": "Web",
        //   "description": "Application uses web based authentication to authenticate user",
        //   "displayName": "Web"
        }
```